shared/src/types/question.ts

```
@@ -0,0 +1,271 @@
+	IMPACT_LIFE_GENERAL,
+	ACHIEVEMENTS_ACHIEVED,
+	ACHIEVEMENTS_NOT_ACHIEVED,
+	HAPPIER,
+	HAPPIER_COMMENT,
+	NOT_HAPPIER_COMMENT,
+	LONG_ENOUGH,
+	RANKING,
+	NOT_EMPLOYED,
+];
+
+export const QUESTIONS_DICTIONARY: Map<String, Question> = new Map(ALL_QUESTIONS.map((obj) => [obj.name, obj]));
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)